requirements_collections.yml

```diff
@@ -0,0 +1,25 @@
+---
+collections:
+  - name: ansible.utils
+    version: 5.1.1
+  - name: community.crypto
+    version: 2.22.0
+  - name: community.docker
+    version: 3.12.1
+  - name: community.general
+    version: 9.4.0
+  - name: community.grafana
+    version: 2.1.0
+  - name: community.mysql
+    version: 3.10.3
+  - name: community.postgresql
+    version: 3.5.0
+  - name: dellemc.os10
+    version: 1.1.1
+  - name: dellemc.openmanage
+    version: 9.6.0
+  - name: kubernetes.core
```

Comment:
So, it looks like you have the Ansible K8s module as a requirement but it's not used in most of the K8s related tasks?

Reply:
@francares python needs to be present on compute nodes to use Kubernetes module on compute node host. Currently, we dont install python on compute nodes just to execute play, hence this module will be used only for play with host as omnia control plane.. 

We install python on compute node for 2 cases only: 1. on login node as it is required for secure login functionality and 2. while deploying vLLM on NVIDIA node as a prereq for that package.

Reply:
You should use K8s and Helm collections from the workstation node where you launch the playbook. This shouldn't be done in each of the control or compute nodes.
Using these collections, they consume K8s APIs and therefore you don't need to be on the cluster nodes to consume the APIs.